app.js

```diff
@@ -4,9 +4,11 @@ const userRouter = require('./routes/userRouter')
 const app =express()
 const port = 3000
 const db =require('./model/index')
+
 require('./model/index')
 
 
+
 app.use(express.json())
 
 app.use('/users',userRouter)
```

Comment:
Use a formatter and linter such as ESLint and Prettier.

Refer: https://dev.to/devdammak/setting-up-eslint-in-your-javascript-project-with-vs-code-2amf

Reply:
ok,